Example/VOKMockData/GET|-auth|Authorization=Token%20I_am_a_token&Content-Language=en.http

```diff
@@ -0,0 +1,3 @@
+HTTP/1.1 202 Accepted
```

Comment:
I wouldn't expect a `202 Accepted` on a `GET` request... but :shrug: doesn't actually matter for the purposes of testing.